src/Runner.php

```diff
@@ -156,25 +164,43 @@ public function start( $target_dir, $opts ) {
 		if ( $existing_PR_branch ) {
 			// Add comment to existing PR with $message
 			$commitSha = exec( 'git rev-parse HEAD', $output_lines, $return_code);
-			$this->addCommitComment( $message, $this->project(), $commitSha );
-			Logger::success( 'Updated pull request with composer.lock changes.' );
+			if ($this->isGitHub()) {
+				$this->addCommitComment( $message, $this->project(), $commitSha );
+				Logger::success( 'Updated pull request with composer.lock changes.' );
+			}
+			elseif ($this->isGitLab()) {
+				$mrNum = $this->findExistingMRNumber();
+				passthru('lab mr note ' . escapeshellarg($mrNum) . ' -m ' . escapeshellarg( $message ));
+				Logger::success( 'Updated merge request with composer.lock changes.' );
+			}
 			return;
 		}
 
-		$cmd = 'hub pull-request -m ' . escapeshellarg( $message );
+		if ($this->isGitLab()) {
+			$cmd = 'lab mr create -m ' . escapeshellarg( $message );
+		}
+		else {
+			$cmd = 'hub pull-request -m ' . escapeshellarg( $message );
+		}
 		Logger::info( $cmd );
 		passthru( $cmd, $return_code );
 		if ( 0 !== $return_code ) {
-			Logger::error( 'Failed to create a pull request with hub.' );
+			Logger::error( 'Failed to create a pull request with CLI tool.' );
 		}
 
 		Logger::success( 'Created pull request with composer.lock changes.' );
 	}
 
 	private function checkExisting() {
-		exec('hub pr list --format="%t%n" --state=open', $output_lines, $return_code);
+		if ($this->isGitLab()) {
+			$command = 'lab mr list';
+		}
+		else {
+			$command = 'hub pr list --format="%t%n" --state=open';
+		}
+		exec($command, $output_lines, $return_code);
 		if ( 0 !== $return_code ) {
-			Logger::error( 'Unable to check for existing pull requests with hub.' );
+			Logger::error( 'Unable to check for existing pull requests with CLI program.' );
```

Comment:
The GiLab merge request language is used elsewhere. We should probably use it here too for consistency.